location/models.py

```diff
@@ -142,6 +142,20 @@ def picture_url(self):
             return self.image
 
 
+class OtherImages(models.Model):
+    apartment = models.ForeignKey(Apartment, default=None, on_delete=models.CASCADE)
+    image = models.ImageField(null=True, blank=True, verbose_name="Image")
+
+    @property
+    def picture_url(self):
+        if not self.image:
+            return None
+        elif "http" not in self.image.url:
+            return self.image.url
+        else:
+            return self.image
+
+
```

Comment:
would it be possible to get some test cases to cover this?

<img width="777" alt="Screen Shot 2019-12-09 at 9 29 49 PM" src="https://user-images.githubusercontent.com/32490273/70489960-23d3fc80-1acb-11ea-8018-d3cfe1ce637f.png">

Reply:
@ab7289 Updated!